src/scpi.c

```diff
@@ -105,23 +105,24 @@ int scpi(char *ip, int port, int timeout, lxi_protocol_t protocol, char *command
 
     // Disconnect
     lxi_disconnect(device);
-
+    free(response);
     return 0;
 
 error_send:
 error_receive:
 
     // Disconnect
     lxi_disconnect(device);
+    free(response);
```

Comment:
Remove this free() else we get a double free error.

Reply:
Indeed missed the fact that each error goto did not have a return, will fix that.